bootstrap_salt/fab_tasks.py

```diff
@@ -32,6 +32,21 @@ def aws(x):
 
 
 @task
+def environment(x):
+    env.environment = str(x).lower()
+
+
+@task
+def application(x):
+    env.application = str(x).lower()
+
+
+@task
+def config(x):
+    env.config = str(x).lower()
```

Comment:
I think people haven't been running the 'setup' task, running against a blank environment..